src/store/metadata/stub/content/editions/persistent-absences-for-pupils_2023-05.json

```diff
@@ -0,0 +1,107 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils/editions",
```

Comment:
I think you should use the full dataset name as there is another dataset - `persistent-absences-for-all-pupils`, to avoid any confusion